runtime/testy/fake_dawn.py

```diff
@@ -2,34 +2,52 @@
 import threading
 import queue
 import time
+import runtime_pb2
+import ansible_pb2
 
 data = [0]
 send_port = 1236
 recv_port = 1235
 dawn_hz = 10
 
+def dawn_packager(data):
+    proto_message = ansible_pb2.DawnData()
+    proto_message.student_code_status = ansible_pb2.DawnData.TELEOP
+    test_gamepad = proto_message.gamepads.add() 
+    test_gamepad.index = 0
+    test_gamepad.axes.append(.5)
+    test_gamepad.buttons.append(True)
+    return proto_message.SerializeToString()
+
 def sender(port, send_queue):
-	host = '127.0.0.1'
-	with socket.socket(socket.AF_INET, socket.SOCK_DGRAM) as s:
-		while True:
-			next_call = time.time()
-			msg = None
-			string = "From Dawn" 
-			b = bytearray()
-			b.extend(map(ord, string))
-			msg = bytes(b)
-			s.sendto(msg, (host, send_port))
-			next_call += 1.0/dawn_hz
-			if next_call > time.time():
-				time.sleep(next_call - time.time())
+    host = '127.0.0.1'
+    with socket.socket(socket.AF_INET, socket.SOCK_DGRAM) as s:
+        while True:
+            next_call = time.time()
+            msg = None
```

Comment:
This line should be removed (because it's redundant with the next line as it is)